quarto_python_formatter.py

```diff
@@ -0,0 +1,71 @@
+# numpydoc ignore=GL08
+#!/usr/bin/env python3
+import re
+import subprocess
+import sys
+import tempfile
+from pathlib import Path
+from typing import Match
+
+
+def format_python_code(code: str) -> str:  # numpydoc ignore=RT01
+    """Format Python code using Black."""
+    try:
+        result = subprocess.run(
+            ["black", "-", "-q", "--line-length", "79"],
```

Comment:
I think it's worth make the formatting call configurable so that things like this can be set in `.pre-commit-config.yaml`

Reply:
addressed in https://github.com/CDCgov/multisignal-epi-inference/pull/265/commits/95efca66f3c3810e13416cc150165f7cf4500b66